src/show/views.py

```diff
@@ -76,3 +77,14 @@ def post(self, request, pk):
             show.save()
 
         return success_response(self.serializer_class(show, context={"request": request}).data)
+
+
+class AddShowToListsView(generics.GenericAPIView):
+
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def post(self, request, pk):
+        data = json.loads(request.body)
+        list_ids = data.get("list_ids")
+        add_show_to_lists(pk, list_ids, request.user)
```

Comment:
there's no point in making this an asynchronous task if you dont use `add_show_to_lists.delay(pk, list_ids..)` which is why you think `request.user` works but it's not actually allowed as a parameter